lib/greed/score.rb

```diff
@@ -0,0 +1,26 @@
+module Greed
+  module Score
+    def calcScore(dice)
+      raise ArgumentError.new unless dice
+      score = 0
+
+      # Create hash of count of each die
+      counts = dice.inject(Hash.new(0)) {|hash, num| hash[num] += 1; hash}
```

Comment:
You got it. The alternative would be defining a start value for each key that you need which doesn't yet exist.